README.md

```diff
@@ -10,6 +10,6 @@ Add the following to your `pipeline.yml`:
 steps:
   - command: ls
     plugins:
-      - elastic/vault-docker-login#v0.5.1:
+      - elastic/vault-docker-login#v0.5.2:
```

Comment:
was this also needed at this time?

Reply:
Yes, this is what was originally blocking the merge.